### ISSUE TITLE
Missing string

### ISSUE THREAD

**Issue by mudrd8mz** [Wunderbyte-GmbH/moodle-local_catquiz#586]
When installing the plugin, I am getting:

```
Invalid get_string() identifier: 'subplugintype_catmodel_plural' or component 'local_catquiz'. Perhaps you are missing 
$string['subplugintype_catmodel_plural'] = ''; in /home/mudrd8mz/www/mdk/m404/moodle/local/catquiz/lang/en/local_catquiz.php?

    line 356 of /lib/classes/string_manager_standard.php: call to debugging()
    line 7028 of /lib/moodlelib.php: call to core_string_manager_standard->get_string()
    line 646 of /lib/classes/plugin_manager.php: call to get_string()
    line 1063 of /admin/renderer.php: call to core\plugin_manager->plugintype_name_plural()
    line 206 of /admin/renderer.php: call to core_admin_renderer->plugins_check_table()
    line 710 of /admin/index.php: call to core_admin_renderer->upgrade_plugin_check_page()
```

**Comment by mudrd8mz** [comment]
This is where the string is to be used:

![image](https://github.com/user-attachments/assets/bb6724e4-95ec-406e-9879-1e486fafe9c4)

**Comment by davidszkiba** [comment]
@mudrd8mz Thanks for checking and opening this issue.
This is fixed in e68abb6ab and included in the new 1.0.3 release
